lca-cli/ops/execute.go

```diff
@@ -48,6 +49,13 @@ func (e *executor) execute(liveLogger io.Writer, root, command string, args ...s
 	return stdoutBytesTrimmed, nil
 }
 
+func (e *executor) executeBash(writer io.Writer, root, command string, args ...string) (string, error) {
```

Comment:
Why we need it? Can you use function from ops?

Reply:
Removing the "bash -c by default" from the chroot altogether, leaving it to the caller to use it in specific cases when absolutely necessary.